Lib/ufo2ft/featureCompiler.py

```diff
@@ -91,6 +95,8 @@ def __init__(self, ufo, ttFont=None, glyphSet=None, **kwargs):
             glyphSet = ufo
         self.glyphSet = OrderedDict((gn, glyphSet[gn]) for gn in glyphOrder)
 
+        self.skipExportGlyphs = skipExportGlyphs
```

Comment:
don't you wanna do the "if skipExportGlyphs is None [then try get it from the font.lib]" dance here as well?

Reply:
oh nevermindm, you already do that later inside feature writer's setContext method